src/sass/_style.scss

```diff
@@ -11,7 +11,7 @@
 // ----------------------------
 // #COLORS
 // ----------------------------
-$font-blue: #126f9a;
+$font-blue: #0e3639;
```

Comment:
Use primary color variable from paragon. Don't hardcode it.

Reply:
Also, this change doesn't make sense. Why does `font-blue` has some color that is not blue? 
You should update the SSO buttons to use `$primary` variable and not change it here. This variable may be used in some other CSS class and changing it here will impact those places too (If its not being used, remove the `font-blue` variable).

Reply:
@zainab-amir this ```$font-blue``` is only used around sso button like color text, border color and background color that is why I have changed at one place instead of multiple places if you still think we need to changed it then we can change it

Reply:
Yes, logically it doesn't make sense to have green or yellow or grey color in "font-blue"